### PR TITLE
Applied the following changes addressed by Cathy for integration

### PR DESCRIPTION
**Pull request by jshaofuturewei**
rename startup.sh to globalschduler-up.sh
Remove the duplicate controller “main/creation and run” codes in cmd/ directory, replace them with the distributor_process.go, dispatcher_process.go and schedule_process.go
Clean up the globalschduler-up.sh “to only include build for all the needed  binaries: cmd/gs_controller, cmd/grpc_server, cmd/scheduler_process, cmd/distributor_process, cmd/dispatcher_process” and “to just invoke gs_controller_manager”. Now all controllers will be started by the gs_controller_manager.go instead of being started manually.

### CODE REVIEW

File: globalscheduler/cmd/dispatcher/dispatcher.go [hunk 1]
```diff
@@ -1,103 +0,0 @@
...
-// client config
-var (
-	flagSet              = flag.NewFlagSet("dispatcher_controller", flag.ExitOnError)
-	dispatchermaster     = flag.String("dispatchermaster", "", "The address of the Kubernetes API server. Overrides any value in kubeconfig. Only required if out-of-cluster.")
-	dispatcherconfig     = flag.String("dispatcherconfig", "/var/run/kubernetes/controller.kubeconfig", "Path to a kubeconfig. Only required if out-of-cluster.")
-	onlyOneSignalHandler = make(chan struct{})
-	shutdownSignals      = []os.Signal{os.Interrupt, syscall.SIGTERM}
-)
-
-func StartDispatcherController() {
```
Comment (CoderKevinZhang) [inline globalscheduler/cmd/dispatcher/dispatcher.go:45]: Is there any reason that we need to delete StartDispatcherController()

Comment (jshaofuturewei) [reply globalscheduler/cmd/dispatcher/dispatcher.go:45]: Cathy asked to keep only process in the cmd folders and let gs-controller-managers to start all the controllers.

**Review by CoderKevinZhang — APPROVED** [review]

File: hack/globalscheduler/globalschduler-up.sh [hunk 2]
```diff
@@ -104,7 +104,7 @@ do
 done
 
 if [ "x${GO_OUT}" == "x" ]; then
-    make -C "${KUBE_ROOT}" WHAT="cmd/kubectl cmd/hyperkube cmd/kube-apiserver cmd/kubelet cmd/kube-proxy cmd/kube-controller-manager globalscheduler/cmd/gs-controllers globalscheduler/cmd/dispatcher/dispatcherprocess globalscheduler/cmd/distributor/distributorprocess"
+    make -C "${KUBE_ROOT}" WHAT="cmd/kubectl cmd/hyperkube cmd/kube-apiserver cmd/kubelet cmd/kube-proxy cmd/kube-controller-manager cmd/kube-scheduler globalscheduler/cmd/gs-controllers globalscheduler/cmd/dispatcher_process  globalscheduler/cmd/distributor_process globalscheduler/cmd/scheduler_process globalscheduler/cmd/grpc-server"
```
Comment (kimeunju108) [inline hack/globalscheduler/globalschduler-up.sh:107]: why don't we use "make all" instead of "make". 
"make all" will build all dependencies.

Comment (jshaofuturewei) [reply hack/globalscheduler/globalschduler-up.sh:107]: Please check the makefile definitions

# Old-skool build tools.
#
# Commonly used targets (see each target for more information):
#   all: Build code.
#   test: Run tests.
#   clean: Clean up.

make all only builds code based on BUILD in your root directory, which are already in the WHAT targets

All the make will build dependencies for your targets.

File: globalscheduler/cmd/distributor_process/BUILD [hunk 3]
```diff
@@ -14,7 +14,7 @@ go_library(
 )
 
 go_binary(
-    name = "process",
+    name = "distributor",
```
Comment (cathyhongzhang) [inline globalscheduler/cmd/distributor_process/BUILD:17]: rename this to distributor_process to be consistent with other parts of the code

**Review by cathyhongzhang — APPROVED** [review]

File: globalscheduler/cmd/distributor_process/BUILD (hunk 3 above)
Comment (jshaofuturewei) [reply globalscheduler/cmd/distributor_process/BUILD]: Updated

**Pull request merged**